docs/pages/tutorial/eas/manage-app-versions.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Manage different app versions
+sidebar_title: Manage app versions
+description: Learn about developer-facing and user-facing app versions and how to manage them automatically.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this chapter, we're going to configure our app to auto-increment the developer-facing app version. Learning about it will be useful before we dive into production build in the next few chapters.
+
+## Difference between developer-facing and user-facing app versions
```

Comment:
Should we also explain why there is a `version` _and_ a build number and the rules around them?

- once you've uploaded an app with a version and build number to apple / google, you can't use the same version & build number combination again
- on android, the build number must always be increasing (so e.g. if you've used 5, you can't use 4 even if you never created a build with 4)
- the idea is that as you're working on e.g. 1.0.0 of your app and need to upload new versions, you increment the build number only until that version is published. Once you publish 1.0.0 you can't upload another build with this version so you have to increment it

Reply:
I've added explanation in context to app stores and relying on the same terminology to help explain things a bit better.